src/status_im/common/home/top_nav/view.cljs

```diff
@@ -37,6 +39,8 @@
      {:avatar-on-press          #(rf/dispatch [:open-modal :settings])
       :scan-on-press            #(rf/dispatch [:open-modal :shell-qr-reader])
       :activity-center-on-press #(rf/dispatch [:activity-center/open])
+      :wallet-latest-update     (when (= (:status wallet-blockchain-status) "down")
+                                  wallet-latest-update)
```

Comment:
We could move this to the let above.

Quick question: Does the status need to be displayed when the user is on the community or chat tab? Because this could give a wrong impression to the user in the context of community and chats.